dhis-2/dhis-api/src/main/java/org/hisp/dhis/keyjsonvalue/KeyJsonValueService.java

```diff
@@ -125,6 +126,9 @@
      */
     void deleteKeyJsonValue( KeyJsonValue entry );
 
+    @Transactional
```

Comment:
I don't think we want Transactional annotations on interfaces, better to have it on the implementation class.

Reply:
that one was auto added by IDE, removed now.